docs/manual/guides/filter.de.md

```diff
@@ -0,0 +1,458 @@
+...
+```
+
+Erstelle dir hierzu in dem von dir unter »Themes« vorgegebenen [Template Verzeichnis](/de/layout/templates/verwaltung/) 
+zwei neues Template basierend auf »`ce_text.html5`« und »`block_searchable.html5`«. 
+
+Beispielsweise als »`ce_text_filter.html5`« und »`block_searchable_filter.html5`« und benutze das neue 
+Template »`ce_text_filter.html5`« in deinen zu filternden Inhaltselementen vom Typ »`Text`«.
+
+
+```php
+// ce_text_filter.html5
+
+<?php $this->extend('block_searchable_filter'); ?>
+
+<?php $this->block('content'); ?>
+
+  <?php if (!$this->addBefore): ?>
+    <?= $this->text ?>
+  <?php endif; ?>
+
+  <?php if ($this->addImage): ?>
+    <?php $this->insert('image', $this->arrData); ?>
+  <?php endif; ?>
+
+  <?php if ($this->addBefore): ?>
+    <?= $this->text ?>
+  <?php endif; ?>
+
+<?php $this->endblock(); ?>
+```
+
+
+```php
+// block_searchable_filter.html5
+
+<?php
+$strDelimiter = "DATA-";
+$strPattern = '/'.$strDelimiter.'(.+?)\b/i';  
+$strDataAttr = "data-category"; 
+$strCSS = $this->class; 
+
+if ( substr_count($strCSS, $strDelimiter) > 0 ) {
+
+  preg_match_all($strPattern, $strCSS, $arrMatches, PREG_PATTERN_ORDER, 0);
+
+  for( $i = 0; $i <= count($arrMatches); $i++) {
+    $strCSS = str_replace($arrMatches[0][$i], "", $strCSS);          
+    $arrMatchedValues[] = $arrMatches[1][$i];      
+  }    
+  $strData = $strDataAttr.'="'.rtrim(implode(", ", $arrMatchedValues), ", ").'"';
+}
+?>
+
+<div class="<?= $strCSS ?> block"<?= $strData ?><?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?>>
+
+  <?php $this->block('headline'); ?>
+    <?php if ($this->headline): ?>
+      <<?= $this->hl ?>><?= $this->headline ?></<?= $this->hl ?>>
+    <?php endif; ?>
+  <?php $this->endblock(); ?>
+
+  <?php $this->block('content'); ?>
+  <?php $this->endblock(); ?>
+
+</div>
+```
+
+{{% notice tip %}}
+Das Script erwartet die Inhalte innerhalb eines HTML Blocks »`<div class="filter-container">...</div>`«. Zur 
+übersichtlicheren Backend Darstellung könntest du die Contao [Accordeon](/de/artikelverwaltung/inhaltselemente/#akkordeon) 
+Elemente »Umschlag Anfang« und »Umschlag Ende« zweck­ent­frem­den. Im Element »Umschlag Anfang« setzt du dann die 
+CSS-Klasse »filter-container« ein.<br><br>
+Weiterhin haben wir einfachheisthalber die Javascript Referenzen direkt im Inhaltselement eingetragen. Alternativ
+könntest du diese auch als [Javascript Asset im Template](/de/layout/templates/assets/) hinterlegen.
+{{% /notice %}}
+
+{{% notice info %}}
+<strong>Vorteil:</strong><br>
+Du bist nicht auf Erweiterungen angewiesen und du hast die vollständige Kontrolle hinsichtlich der Umsetzung 
+und der Pflege. Bei Contao Updates müssen u. U. lediglich mögliche Änderungen der Core Templates berücksichtigt werden.
+<br><br>
+<strong>Nachteil:</strong><br>
+Für Template Anpassungen in dieser Form sind zumindest rudimentäre PHP Kenntnisse vorteilhaft. Die 
```

Comment:
"Nachteil ...", daß etwas "vorteilhaft" ist klingt komisch. 

"... in dieser Form sind zumindest rudimentäre PHP Kenntnisse nötig."

Reply:
Done